core/src/components/modal/test/sheet/modal.e2e.ts

```diff
@@ -195,3 +195,111 @@ test.describe('sheet modal: setting the breakpoint', () => {
     expect(updatedBreakpoint).toBe(0.5);
   });
 });
+
+test.describe('sheet modal: handleBehavior cycle: focus trap', () => {
+  test.beforeEach(async ({ page, browserName }, testInfo) => {
+    test.skip(testInfo.project.metadata.rtl === true, 'This does not test LTR vs. RTL layout.');
+    test.skip(browserName === 'firefox', 'Firefox incorrectly allows keyboard focus to move to ion-content');
+    test.skip(browserName === 'webkit', 'Safari incorrectly allows keyboard focus to move to the slotted content');
```

Comment:
Do we need tech debt tickets for either of these? The `webkit` one almost just sounds like a bug, but there may be context I'm missing.

Reply:
I'm not sure there is much we can do other than file issues with browser vendors. This the same issue as https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/modal/test/basic/modal.e2e.ts#L8.

The problem isn't that focus trapping isn't working, it's that focus is not being moved inside of the container to where we expect it to be. We could test in the negative (I.e. check that body is not focused), but I also worry we may miss some edge cases with that.

Reply:
Alternatively, we could check if the focused element exists inside of the container. That would be a bit of custom code, but it could be more reliable.

Reply:
If it's the same issue, why does the basic test work in Webkit but this one doesn't? Just a different HTML layout? 🤔

Reply:
On second thought... do we even need this test? The focus trapping tests are handled in https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/modal/test/basic/modal.e2e.ts#L5. Focus trapping in sheet modals should not even be enabled https://github.com/ionic-team/ionic-framework/issues/25147

Reply:
Sounds good. Not sure what would cause the inconsistency. FWIW Chrome + Firefox did focus the handle when pressing Tab with the sheet open. Safari moved focus to the Dismiss Modal button, but pressing Shift + Tab did focus the handle. Maybe a bug with our focus trap util?